packages/tradershub/src/flows/RealAccountSIgnup/SignupWizard/Actions/index.tsx

```diff
@@ -14,37 +15,40 @@ type TActions = {
  * Intended to be used as a child component of the Formik component.
  * @param {Object} props - React props object
  * @param {boolean} [props.submitDisabled] - A boolean that determines whether the Next button is disabled
+ * @param {boolean} [props.isSubmitBtnLoading] - A boolean that determines whether the Next button is in a loading state
  * @example
  * return (
  *     <Actions submitDisabled />
  * );
  */
 
-const Actions = ({ submitDisabled = false }: TActions) => {
+const Actions = ({ submitDisabled = false, isSubmitBtnLoading = false }: TActions) => {
     const {
         helpers: { canGoToNextStep, canGoToPrevStep, goToPrevStep },
     } = useSignupWizardContext();
-    const { handleSubmit } = useFormikContext();
+    const { isSubmitting } = useFormikContext();
     const { isDesktop } = useDevice();
 
     return (
         <div>
             <Divider />
-            <ButtonGroup className='flex-row justify-end p-24'>
+            <ButtonGroup className='p-24 sm:flex-row md:flex-row md:justify-end'>
```

Comment:
Because of how ButtonGroup is implemented, we have to specify the breakpoints. Because clsx doesnt merge styles, will remove these once we use tailwind-merge